### PR TITLE
Update talks.mdx

### PR DESCRIPTION
**Pull request by AdamPrendergast**
Added conference talk for 'Codegarden 2024: Bake, Don't Fry - Astro & The Content Delivery API'

#### Description (required)

I am proposing the addition of my conference talk, "Bake, Don't Fry - Astro & The Content Delivery API," presented at Codegarden 2024, to the Astro documentation under the list of community talks.

I also just presented it at the Umbraco US Festival in Chicago last week

In the talk, I explore what Astro is and how it can be integrated with the Umbraco CMS Content Delivery API to create a CMS-powered static site. Although the talk originates from an Umbraco-centric context, it offers broader value by covering TypeScript type definition generation and showcasing Astro examples that range from simple 'Hello World' implementations to more advanced integrations.

It would be fantastic to see this talk shared with the wider Astro community, as I believe it can provide useful insights for developers working with Astro, particularly in CMS contexts.

Any feedback or suggestions regarding the talk’s inclusion (or if it would be better placed in another resource) would be greatly appreciated.

I would also greatly welcome feedback on the talk itself 🙂

Thank you very much for your time and consideration, and I look forward to hearing your thoughts! 🙌

Discord user: Prenders

### CODE REVIEW

**Comment by yanthomasdev** [comment]
Thanks @AdamPrendergast, that's a lovely talk. I believe this resource should instead be in the 
["Community Resources" section of the Umbraco guide](https://docs.astro.build/en/guides/cms/umbraco/#community-resources), since it's a talk geared towards integrating Astro with Umbraco. Once you make that change, I can quickly go ahead and merge it 🙌

**Comment by sarah11918** [comment]
Hi @AdamPrendergast ! Just checking in here whether you're still up for moving this link to the Umbraco guide itself!

The page you've listed is mostly for very high-level introductions to Astro, and yours is more specific to a task, so that's not quite the right place for it. But we'd love to add it to the Umbraco guide! And I notice you'll be giving this talk again at the end of the month? Good luck! We've really enjoyed seeing the Umbraco community embrace and get into Astro! 🚀

**Comment by AdamPrendergast** [comment]
Hey Sarah & Yan,

Thanks for checking in - totally agree with placing it in the Umbraco docs
section. I had a few projects needing my attention. so have not got back to
updating just yet. Will get it sorted very soon!

Thanks again, and yes, we are loving Astro at Umbraco 🥰

**Comment by AdamPrendergast** [comment]
Hi all,

I have updated the PR to move the link to the talk to the Umbraco guide.

Thanks!

Adam

File: src/content/docs/en/guides/cms/umbraco.mdx [hunk 1]
```diff
@@ -229,3 +229,4 @@ This method is described in more detail in this [blog post showing how to config
 - [Generating a TypeScript OpenAPI client from Umbraco's Content Delivery API - Rick Butterfield](https://rickbutterfield.dev/blog/typescript-openapi-umbraco-content-delivery/)
 - [Jamstack For Free With Azure And CloudFlare - Kenn Jacobsen](https://kjac.dev/posts/jamstack-for-free-with-azure-and-cloudflare/)
 - [Quick n' dirty blog with Astro and Umbraco - Kenn Jacobsen](https://kjac.dev/posts/quick-n-dirty-blog-with-astro-and-umbraco/)
+- [Talk: Bake, Don't Fry - Astro & The Content Delivery API](https://www.youtube.com/watch?v=zNxqI25dtx4) by Adam Prendergast, Codegarden 2024
```
Comment (sarah11918) [inline src/content/docs/en/guides/cms/umbraco.mdx:232]:
```suggestion
- [Talk: Bake, Don't Fry - Astro & The Content Delivery API - Adam Prendergast](https://www.youtube.com/watch?v=zNxqI25dtx4)
```
Just formatting it like the others here for consistency!

**Review by sarah11918 — APPROVED** [review]
Thanks @AdamPrendergast ! I'm just updating the link style to match the other ones, but this is great! Thanks for spreading the word about Astro, and welcome to Team Docs! 🥳

**Review by yanthomasdev — APPROVED** [review]
LGTM! 🫡

**Pull request merged**